config/api.yml

```diff
@@ -3233,6 +3233,10 @@
       :post:
       - :name: cancel
         :identifier: miq_request_control
+      - :name: deliver
+        :identifier: miq_request_control
+      - :name: execute
+        :identifier: miq_request_control
```

Comment:
I'm not sure here - This would give users with the ability to cancel the requests to also execute and deliver, which is really a system level task - I wonder if this should be super admin only or something 🤔

Reply:
@Fryguy did you just say `service_admin`?

Reply:
this will be using the authentication record that workflow/floe is passing. (api_user)

@agrare Do you typically pass superadmin to floe?
Do you feel that is what customers should do?

Reply:
I had imagined that the user ordering the service would be the one posting to the API, an admin would create the workflows and map what credentials would be used but at order time it would be the unprivileged user account

Reply:
Most of these tasks will talk to the API and want to act as the requesting user.

But there are requests that need to be super admin.

This endpoint is basically saying "All the workflows have been run. All the variables have been figured out. Let's talk with vmware and make the actual request happen"

That should only be available from a super admin.
The current vmware specific workflow has:
- vmware admin auth
- miq requestor user auth

So it would make sense to have:
- miq admin auth (to make provider privileged requests)
- miq requestor user auth (to do stuff as user privileges)

Just not sure the best way to pass the admin auth

Reply:
I'm lost now about who and when - let's discuss over voice and maybe draw it out.